.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
remove all .idea files from your commit - they should not be added to master.
these are pycharm config files